pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -254,21 +257,21 @@ void ATA::CreateHDDinfo(u64 sizeSectors)
 	// clang-format off
 	WriteUInt16(identifyData, &index, static_cast<u16>(
 		/*(1 << 8) |						//SET MAX */
-		((lba48Supported ? 1 : 0) << 10) |	//Fixed
+		(lba48Supported << 10) |			//Fixed
```

Comment:
Interesting how Codacy didn't flag this before

Reply:
But this is not an actual error, this is user defined value